### ISSUE TITLE
Issue with Experimental Result Reproduction

### ISSUE THREAD

**Issue by bestyou123** [wangchen1801/FPD#35]
Dear author, I have attempted to reproduce the results from your paper. During Base Training on Split1, my Base AP50 only reached 77.0, whereas the Base AP50 for the base training provided in your GitHub repository is 79.0. I used the exact same configuration file as yours, trained for 20,000 epochs, and set the learning rate to 0.005. Could there be any step or detail I missed?
I sincerely look forward to your reply. Thank you very much!

**Comment by wangchen1801** [comment]
May I ask how many GPUs you used? It would also be very helpful if you could share the training script you used.

Thanks!

**Comment by bestyou123** [comment]
用了一个gpu，请问是要用两个gpu吗



---Original---
From: ***@***.***&gt;
Date: Fri, Nov 21, 2025 21:03 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction(Issue #35)


wangchen1801 left a comment (wangchen1801/FPD#35)
 
May I ask how many GPUs you used? It would also be very helpful if you could share the training script you used.
 
Thanks!

**Comment by wangchen1801** [comment]
对的，原实验用了两张显卡，单卡性能会稍差一些



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2025年11月21日 21:29 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction (Issue #35) |
bestyou123 left a comment (wangchen1801/FPD#35)
用了一个gpu，请问是要用两个gpu吗



---Original---
From: ***@***.***&gt;
Date: Fri, Nov 21, 2025 21:03 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction(Issue #35)


wangchen1801 left a comment (wangchen1801/FPD#35)

May I ask how many GPUs you used? It would also be very helpful if you could share the training script you used.

Thanks!

**Comment by bestyou123** [comment]
感谢您的回复，我现在用双卡我们的服务器程序老是崩溃，我想问一下如果用单卡想尽可能的提高一些效果，是否可以尝试增加训练轮次等策略，期待您的解答，谢谢



---Original---
From: ***@***.***&gt;
Date: Fri, Nov 21, 2025 21:41 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction(Issue #35)


wangchen1801 left a comment (wangchen1801/FPD#35)
 对的，原实验用了两张显卡，单卡性能会稍差一些
 
 
 
 ---- 回复的原邮件 ----
 | 发件人 | ***@***.***&gt; |
 | 日期 | 2025年11月21日 21:29 |
 | 收件人 | ***@***.***&gt; |
 | 抄送至 | ***@***.***&gt;***@***.***&gt; |
 | 主题 | Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction (Issue #35) |
 bestyou123 left a comment (wangchen1801/FPD#35)
 用了一个gpu，请问是要用两个gpu吗
 
 
 
 ---Original---
 From: ***@***.***&amp;gt;
 Date: Fri, Nov 21, 2025 21:03 PM
 To: ***@***.***&amp;gt;;
 Cc: ***@***.******@***.***&amp;gt;;
 Subject: Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction(Issue #35)
 
 
 wangchen1801 left a comment (wangchen1801/FPD#35)
 
 May I ask how many GPUs you used? It would also be very helpful if you could share the training script you used.
 
 Thanks!

**Comment by wangchen1801** [comment]
可以试试增大batch size



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2025年11月21日 21:45 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction (Issue #35) |
bestyou123 left a comment (wangchen1801/FPD#35)
感谢您的回复，我现在用双卡我们的服务器程序老是崩溃，我想问一下如果用单卡想尽可能的提高一些效果，是否可以尝试增加训练轮次等策略，期待您的解答，谢谢



---Original---
From: ***@***.***&gt;
Date: Fri, Nov 21, 2025 21:41 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction(Issue #35)


wangchen1801 left a comment (wangchen1801/FPD#35)
对的，原实验用了两张显卡，单卡性能会稍差一些



---- 回复的原邮件 ----
| 发件人 | ***@***.***&gt; |
| 日期 | 2025年11月21日 21:29 |
| 收件人 | ***@***.***&gt; |
| 抄送至 | ***@***.***&gt;***@***.***&gt; |
| 主题 | Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction (Issue #35) |
bestyou123 left a comment (wangchen1801/FPD#35)
用了一个gpu，请问是要用两个gpu吗



---Original---
From: ***@***.***&amp;gt;
Date: Fri, Nov 21, 2025 21:03 PM
To: ***@***.***&amp;gt;;
Cc: ***@***.******@***.***&amp;gt;;
Subject: Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction(Issue #35)


wangchen1801 left a comment (wangchen1801/FPD#35)

May I ask how many GPUs you used? It would also be very helpful if you could share the training script you used.

Thanks!

**Comment by bestyou123** [comment]
好的 感谢您的回复



---Original---
From: ***@***.***&gt;
Date: Fri, Nov 21, 2025 21:51 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction(Issue #35)


wangchen1801 left a comment (wangchen1801/FPD#35)
 可以试试增大batch size
 
 
 
 ---- 回复的原邮件 ----
 | 发件人 | ***@***.***&gt; |
 | 日期 | 2025年11月21日 21:45 |
 | 收件人 | ***@***.***&gt; |
 | 抄送至 | ***@***.***&gt;***@***.***&gt; |
 | 主题 | Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction (Issue #35) |
 bestyou123 left a comment (wangchen1801/FPD#35)
 感谢您的回复，我现在用双卡我们的服务器程序老是崩溃，我想问一下如果用单卡想尽可能的提高一些效果，是否可以尝试增加训练轮次等策略，期待您的解答，谢谢
 
 
 
 ---Original---
 From: ***@***.***&amp;gt;
 Date: Fri, Nov 21, 2025 21:41 PM
 To: ***@***.***&amp;gt;;
 Cc: ***@***.******@***.***&amp;gt;;
 Subject: Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction(Issue #35)
 
 
 wangchen1801 left a comment (wangchen1801/FPD#35)
 对的，原实验用了两张显卡，单卡性能会稍差一些
 
 
 
 ---- 回复的原邮件 ----
 | 发件人 | ***@***.***&amp;gt; |
 | 日期 | 2025年11月21日 21:29 |
 | 收件人 | ***@***.***&amp;gt; |
 | 抄送至 | ***@***.***&amp;gt;***@***.***&amp;gt; |
 | 主题 | Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction (Issue #35) |
 bestyou123 left a comment (wangchen1801/FPD#35)
 用了一个gpu，请问是要用两个gpu吗
 
 
 
 ---Original---
 From: ***@***.***&amp;amp;gt;
 Date: Fri, Nov 21, 2025 21:03 PM
 To: ***@***.***&amp;amp;gt;;
 Cc: ***@***.******@***.***&amp;amp;gt;;
 Subject: Re: [wangchen1801/FPD] Issue with Experimental Result Reproduction(Issue #35)
 
 
 wangchen1801 left a comment (wangchen1801/FPD#35)
 
 May I ask how many GPUs you used? It would also be very helpful if you could share the training script you used.
 
 Thanks!
